.github/workflows/tagPublish.yml

```diff
@@ -66,11 +69,28 @@ jobs:
       - name: build source
         id: build_source
         run: python -u .github/workflows/build_source.py
+      - name: mint API token
+        id: mint-token
+        shell: bash
+        run: |
+            # retrieve the ambient OIDC token
```

Comment:
Add a comment linking to where this documentation was found

Reply:
Done